src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php

```diff
@@ -84,6 +84,16 @@ public function getContentPath($route, $webspaceKey = null, $locale = null, $dom
             $domain,
             $scheme
         );
+        if ($allDomains && !$url) {
```

Comment:
Then the name is very misleading. What about `$withoudDomain`? Is it always delivered without the domain if this variables is set to true? Does it contain the resource locator prefix?